src/dataloader.py

```diff
@@ -7,86 +7,34 @@
 import os
 import pydicom
 import pandas as pd
+import PIL
```

Comment:
What's this PIL thing all about?

Reply:
This is the medicine one needs to take to push past DICOM standards.